webpack.config.js

```diff
@@ -18,7 +20,7 @@ const htmlWebpackPlugin = new HtmlWebpackPlugin({
 module.exports =(env)=> {
     const isProduction = env === 'production';
     return{
-        entry: SRC_DIR + '/index.js',
+        entry: ['babel-polyfill',SRC_DIR + '/index.js'],
```

Comment:
Are you using this `babel-polyfill` to fix the error `ReferenceError: regeneratorRuntime is not defined`? 

If so, I think there is a cleaner way to do it...

Reply:
yes!

Reply:
@irfiacre As we discussed with @kabundege and @benshidanny11 in this thread: https://stackup-workspace.slack.com/archives/GV2N0HZ4G/p1591213421012700?thread_ts=1591213290.012000&cid=GV2N0HZ4G

Here is what you can do instead:

- Install the Babel plugin "Transform Runtime"... `npm install --save @babel/plugin-transform-runtime` 
- Include this plugin in the `.babelrc` (see my file below)

```
{
	"presets": ["@babel/preset-env", "@babel/preset-react"],
	"plugins": [
		"transform-class-properties",
		"@babel/plugin-proposal-object-rest-spread",
		"@babel/plugin-transform-runtime"
	]
}
```
- You can now remove the 'babel-polyfill' from Webpack config file.

I believe this is cleaner because you put the setup logic for Babel inside the Babel config file, instead of mixing it with the Webpack config. This keeps Babel and Webpack separate in terms of configuration.

Hope this helps.